db/models.py

```diff
@@ -1 +1,40 @@
 from django.db import models
+from django.db.models import DateTimeField
```

Comment:
Consider using `models.DateTimeField` directly instead of importing `DateTimeField` separately. This keeps the code cleaner and more consistent with the rest of the field definitions.